components/dialog/dialog-mixin.js

```diff
@@ -18,6 +18,17 @@ if (window.D2L.DialogMixin.preferNative === undefined) {
 	window.D2L.DialogMixin.preferNative = true;
 }
 
+let ifrauDialogService;
+
+async function getIfrauDialogService() {
+	if (ifrauDialogService === undefined && window.ifrauclient) {
```

Comment:
This is a minor point, but I would tend to just check
```javascript
if (!ifrauDialogService && window.ifrauclient)
```
Partly out of laziness, but in this case, I would expect to treat `null` and `undefined` the same... and create a new service in such a case, so omitting the explicit check seems more resilient.